code/game/machinery/doors/airlock.dm

```diff
@@ -802,6 +807,17 @@
 		update_appearance()
 	return TOOL_ACT_TOOLTYPE_SUCCESS
 
+/obj/machinery/door/airlock/wrench_act(mob/living/user, obj/item/tool)
+	if(!locked)
+		return
+	if(istype(tool, /obj/item/wrench/bolter))
+		balloon_alert(user, "raising bolts...")
+		if(!do_after(user, 5 SECONDS, src))
+			return FALSE
+		unbolt()
```

Comment:
Should probably have it check for the panel being open, otherwise you can just use this on the vault and walk in

Reply:
Good point, I will fix it.